pkg/controller/instance/instance_controller.go

```diff
@@ -305,57 +305,42 @@ func instanceEventFilter(mgr manager.Manager) predicate.Funcs {
 	}
 }
 
-func createPlan(r *ReconcileInstance, planName string, instance *kudov1alpha1.Instance) error {
+func createPlan(c client.Client, r record.EventRecorder, scheme *runtime.Scheme, planName string, instance *kudov1alpha1.Instance) error {
```

Comment:
This method should be called `createPlanAndUpdateInstance` or even `createPlanAndUpdateInstanceActivePlanReerence` :P